# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/VectorImage.cpp
# Commit: 2e365eaea75b
# Full Hash: 2e365eaea75b925b2b5ff621634ba8f7d7fec00b
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-22 04:04:03
# Regressor Bug: 1568950
# File Overlap Count: 1
# Description:
#   Bug 1568950 - Part 5: Remove nsIDOMWindowUtils::forceUseCounterFlush; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42818
# ==============================================================================

diff -r 1c1c4ca0bc14 -r 2e365eaea75b image/VectorImage.cpp
--- a/image/VectorImage.cpp	Wed Aug 21 10:29:14 2019 +0000
+++ b/image/VectorImage.cpp	Wed Aug 21 12:52:19 2019 +0000
@@ -1509,12 +1509,6 @@
   }
 }
 
-void VectorImage::ReportUseCounters() {
-  if (Document* doc = mSVGDocumentWrapper->GetDocument()) {
-    doc->ReportUseCounters();
-  }
-}
-
 nsIntSize VectorImage::OptimalImageSizeForDest(const gfxSize& aDest,
                                                uint32_t aWhichFrame,
                                                SamplingFilter aSamplingFilter,